### ISSUE TITLE
NVDA on Google Chrome / Mircrosoft Edge randomly fails when i press Enter from Keyboard on button element

### ISSUE THREAD

**Issue by YogendraJainDTDL** [nvaccess/nvda#17367]
An issue occurs when a user navigates back to a previous page using the browser's back functionality (history.go(-1)), and then tries to interact with a button on the newly loaded page (such as pressing "Enter"). Sometimes, the button press results in an error with no action being performed. This behavior is inconsistent and happens intermittently, sometimes working as expected and other times not.

When the user presses "Enter" on a button after navigating back, the following error is observed in the console:

```
ERROR executing script: <bound method ...> with gesture 'enter' Traceback (most recent call last): 
File "scriptHandler.py", line 300, in executeScript 
File "browseMode.py", line 691, in script_activatePosition 
File "browseMode.py", line 706, in _focusLastFocusableObject 
File "virtualBuffers/gecko_ia2.py", line 357, in _shouldSetFocusToObj 
AttributeError: 'NoneType' object has no attribute 'role'
```

Steps to Reproduce:

1. On a page with a list of 4 options (each option is a button inside a `<ul><li><button /></li></ul>`). click on an option via the keyboard [Press Enter].

2. After being redirected to the selected option’s page, click the back button via the keyboard [Press Enter] (history.go(-1)) to return to the previous page.

3. After navigating back, click on one of the options again, but nothing happens, and the above error is logged.

Note: The issue is intermittent, sometimes the interaction works as expected, and other times the error occurs.

and it starts working if I toogle the Mode means Pres Insert + Spacebar .

### NVDA logs, crash dumps and other attachments:

![NVDA Log Viewer 11_5_2024 6_26_19 PM](https://github.com/user-attachments/assets/b44e93bd-de8f-43fb-a399-5846d09fbde9)

#### NVDA installed/portable/running from source:
NVDA installed

#### NVDA version:
2024.4

#### Windows version:

Windows 10

#### browser Details
Google Crome - Version 130.0.6723.92 
Microsoft Edge - 130.0.2849..68
Firefox - it is working Fine.

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If NVDA add-ons are disabled, is your problem still occurring?

Yes

**Comment by nvdaes** [comment]
Hi @YogendraJainDTDL.
Thanks for reporting this. Seems that the attached file is a screenshot.
Many developers are blind and we will appreciate if a description of images or text files are also provided.
You can follow these [instruction to attach a log file](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps).
Also, I think it would be useful if you can provide links to sample webpages to reproduce this issue, especially considering that this happens intermitently.
Hope this helps.

**Comment by YogendraJainDTDL** [comment]
@nvdaes Please find nvda log details

[nvda.log](https://github.com/user-attachments/files/17643459/nvda.log)

**Comment by Adriani90** [comment]
Please run the COM registration fixing tool in NVDA tool menu. Are still having this issue after that?

**Comment by YogendraJainDTDL** [comment]
@Adriani90 Yes, Tried COM registration fixing tool.  still facing this issue

**Comment by nvdaes** [comment]
Thanks. Also, if you are comfortable modifying and restoring NVDA settings, you may try to see if this happens when UIA is enabled for Chrome and Edge, in advanced settings.
I don't know if this may be relevant or not, but you may want to test it.

**Comment by seanbudd** [comment]
Welcome @YogendraJainDTDL 

We are unable to process this issue as it stands.

To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/)

Kind Regards,
NV Access Software Developers

**Comment by Adriani90** [comment]
@YogendraJainDTDL are you still having this issue in NVDA 2025.1 Beta?
Can you please turn on UIA in Edge and Chromium in NVDA advanced settings? Does this still occur then?
Also please provide a minimal test sample if possible.
I am enclined to close this as a duplicate of #17398 which contains the same behavior and same error.

**Comment by seanbudd** [comment]
Closing as we haven't received the required information. Please open a new issue with a minimal code sample
